index.html

```diff
@@ -124,16 +124,18 @@
         [[!CSSOM-VIEW]] to allow document scripts to inform the browser that
         they prefer to keep the device screen awake:
       </p>
-      <dl title="partial interface Screen" class="idl">
-        <dt>attribute boolean keepAwake</dt>
-        <dd>
-          Reflects the state of the <a>wake lock</a> <a>request
-          flag</a>.
-          Document script can set this attribute to <code>true</code> to
-          <a>request</a> wake lock or to <code>false</code> to
-          <a>cancel</a> wake lock request.
-        </dd>
-      </dl>
+      <pre class="idl">
+        partial interface Screen {
+          attribute boolean keepAwake;
+        };
+      </pre>
+      <p>
+        The attribute <dfn>Screen.keepAwake</dfn> reflects the state of the
```

Comment:
Since `keepAwake` is not `static`, `Screen.keepAwake` is not appropriate. Probably just "The `keepAwake` attribute", or optionally "The `keepAwake` attribute of the `Screen` interface".